metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/rest/execution/details/PluginView.java

```diff
@@ -0,0 +1,100 @@
+package eu.europeana.metis.core.rest.execution.details;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import eu.europeana.metis.CommonStringValues;
+import eu.europeana.metis.core.workflow.plugins.AbstractExecutablePlugin;
+import eu.europeana.metis.core.workflow.plugins.AbstractMetisPlugin;
+import eu.europeana.metis.core.workflow.plugins.DataStatus;
+import eu.europeana.metis.core.workflow.plugins.PluginStatus;
+import eu.europeana.metis.core.workflow.plugins.PluginType;
+import java.util.Date;
+
+public class PluginView {
+
+  private final PluginType pluginType;
+  private final String id;
+  private final PluginStatus pluginStatus;
+  private final DataStatus dataStatus;
+  private final String failMessage;
+  @JsonFormat(pattern = CommonStringValues.DATE_FORMAT)
+  private final Date startedDate;
+  @JsonFormat(pattern = CommonStringValues.DATE_FORMAT)
+  private final Date updatedDate;
+  @JsonFormat(pattern = CommonStringValues.DATE_FORMAT)
+  private final Date finishedDate;
+  private final String externalTaskId;
+  private final PluginProgressView executionProgress;
```

Comment:
I think we should rename this too to match the type name, what do you think?

Reply:
The problem is that this then requires frontend changes... I was trying to keep everything as much the same as possible for the frontend.

Reply:
Then fine for now but I think we need to rethink the names of those two classes..
Having the same name in two different packages can be confusing.